race_transport.go

```diff
@@ -37,6 +38,8 @@ type namedRoundTripper struct {
 	name string
 }
 
+const ampMaxContentLength = 6000
```

Comment:
I really think this should be part of a new interface for all transports.

I.e. something like a `KindlingTransport` interface that would have something like:

```
MaxLength() int
ConnectedRoundTripper(...) http.RoundTripper
```

Reply:
Added [cb4f44b](https://github.com/getlantern/kindling/pull/25/commits/cb4f44b3e3756640b9b4e6b44f4776fb502826aa)